### DISCUSSION TITLE
[generative art] how to group inscriptions into a collection?

### DISCUSSION THREAD

**Discussion by punk3700** [ordinals/ord#1591] [General]
we're launching a long-form gen art collection this week onto ordinals. super excited!  

ord allows us to upload a single file (200kb) and keep the code 100% on-chain with no external dependencies.  we're using the inscription id as the random seed to generate various traits for the artwork.

question: is there a way to group the outputs into a collection (instead of just a bunch of links on the explorers)?  

thanks!

**Comment by casey** [comment]
See #783 for details. This is planned, but we're underwater so will take a while. My advice is to not wait for it, and discuss with other people doing collections. A few ad-hoc community standards have emerged, and although they may not get official support, they are good ways of making sure users understand which things are and aren't in your collection.
